doc/blog/201708-Importing-SL-Toolkit-part-1.md

```diff
@@ -0,0 +1,122 @@
+
+This method can be replaced by the `ItemsControl.Items.VectorChanged` event, but not completely. `ItemsControl` in Silverlight was based on `ObservableCollection` which provided the `NotifyCollectionChangedAction` property, whereas the `ItemsCollection` in UWP is based on `ObservableVector`. This new implementation notably does not provide the `Replace`
+action and it's raising `ItemRemoved` then `ItemInserted` instead.
+
+In this case, we must remove the part that dealt with `NotifyCollectionChangedAction.Replace`, keeping 
+only `Remove`, `Reset` and `Insert`. The new API also does not provide the items being notified for, which means we
+have to use the `Items` property directly instead.
+
+### Temporarily commenting out incompatible features
+
+In other cases, the APIs are significantly different and we'll for now comment those out for the sake of chewing a comfortable piece of code. Parts of this include keyboard support, parts of the mouse support, localization and Peer Automation related support.
+
+UWP provides support for those features, not as virtual methods but rather as events that are not directly compatible with the Silverlight implementation. We will take a look at those features in a later part of the series.
+
+### Adjusting the XAML
+
+Another part of the migration to UWP is the adjustment of the XAML. The syntax is the same, but parts differ :
+
+- The custom namespace syntax is now using `using:` instead of `clr-namespace:`, and in many cases simply replacing one by the other is enough
+- The `VisualStateManager` related classes are now in the default xml namespace, meaning that the `vsm:` namespace is not required anymore.
+- The `system` namespace is now part of the `x:` namespace
+
+We'll also comment out the support for `Cursor` adjustments, as the property is not directly mapped to UWP.
+
+## Creating the sample app
+
+To be able to test the imported code, we can create a sample application using [the Uno Platform VS Addin](), reference our imported project and add a simple sample like this one:
+
+```xml
+<controls:TreeView Margin="5">
+	<controls:TreeViewItem Header="Controls">
+	<controls:TreeViewItem Header="AutoCompleteBox">
+		<controls:TreeViewItem Header="Properties" />
+	</controls:TreeViewItem>
+	<controls:TreeViewItem Header="Expander" />
+		<controls:TreeViewItem Header="NumericUpDown" />
+	</controls:TreeViewItem>
+	<controls:TreeViewItem Header="Layout">
+		<controls:TreeViewItem Header="DockPanel" />
+		<controls:TreeViewItem Header="WrapPanel" />
+		<controls:TreeViewItem Header="Viewbox" />
+	</controls:TreeViewItem>
+	<controls:TreeViewItem Header="Charting">
+		<controls:TreeViewItem Header="ColumnSeries" />
+		<controls:TreeViewItem Header="LineSeries" />
+		<controls:TreeViewItem Header="PieSeries" />
+	</controls:TreeViewItem>
+</controls:TreeView>
+```
+
+and test it first on Windows, then iOS, Android and WebAssembly.
+
+## Next steps
+
+When running the sample, a few things stand out: 
```

Comment:
Maybe highlight the good and the bad? (eg https://stackoverflow.com/a/41247934/1902058) (or ASCII ticks and crosses)